README.md

```diff
@@ -39,3 +39,173 @@ scalability, flexibility and traceability. Eiffel is based on the concept of dec
 messaging, both to drive the continuous integration and delivery system and to document it.
 
 Visit [Eiffel Community](https://eiffel-community.github.io) to get started and get involved.
+
```

Comment:
This file is just a duplication of the about.md could there not just be description and a link to about.md from here?

Reply:
Yes, I can do that, the formatting was rendered worse in github but I used the github formatting in the about.md instead and it looks a bit better now.